modules/nf-core/glimpse/phase/main.nf

```diff
@@ -0,0 +1,86 @@
+process GLIMPSE_PHASE {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "bioconda::glimpse-bio=1.1.1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/glimpse-bio:1.1.1--h2ce4488_2':
+        'quay.io/biocontainers/glimpse-bio:1.1.1--hce55b13_1' }"
+
+    input:
+        tuple val(meta), path(input), path(inp_index)
+        path(map)
+        path(samples_file)
+        tuple val(ref), path(reference), path(ref_index)
+        path(region_file)
+        val(input_region)
+        val(output_region)
```

Comment:
These values should be given through `task.ext.args`

Reply:
Should they ?
As they are necessary arguments, I thought they needed to be specified ?